docs/src/components/BrandedTile.tsx

```diff
@@ -21,16 +21,21 @@ interface Props {
   children: React.ReactNode;
 }
 
-const StyledWrapper = styled.div<{ primary: string }>`
+const StyledWrapper = styled.a<{ primary: string }>`
```

Comment:
This change affects other tiles that aren't links, like those on the [Roadmap](https://8af2731c.orbit.pages.dev/roadmap/). Could the element be decided based on props or something else?

Reply:
Also, this leads to invalid DOM nesting with `<a>` elements inside other `<a>` elements. So if this is an `<a>` element, the link in the bottom right of the tile should change.

Reply:
Fixed both problems. The second one by turning nested links and buttons into click-through spans and divs.